recipes/sdl_image/3.x/conanfile.py

```diff
@@ -0,0 +1,140 @@
+        "with_avif": [True, False],
+        "with_jxl": [True, False],
+        "with_imageio": [True, False],
+        "with_wic": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_libjpeg": True,
+        "with_libtiff": True,
+        "with_libpng": True,
+        "with_libwebp": True,
+        "with_avif": True,
+        "with_jxl": False,
+        "with_imageio": True,
+        "with_wic": False,
+    }
+    implements = ["auto_shared_fpic"]
+    languages = "C"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+        if not is_apple_os(self):
+            del self.options.with_imageio
+        if self.settings.os != "Windows":
+            del self.options.with_wic
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("sdl/[>=3.2.20 <4]", transitive_headers=True)
+        if self.options.with_libtiff:
+            self.requires("libtiff/4.6.0")
+        if self.options.with_libjpeg:
+            self.requires("libjpeg/[>=9e]")
+        if self.options.with_libpng:
+            self.requires("libpng/[>=1.6 <2]")
+        if self.options.with_libwebp:
+            self.requires("libwebp/[>=1.3.2 <2]")
+        if self.options.with_avif:
+            self.requires("libavif/[>=1.0.1 <2]")
+        if self.options.with_jxl:
+            self.requires("libjxl/0.11.1")
+
+    def validate(self):
+        if self.options.shared and not self.dependencies["sdl"].options.shared:
+            raise ConanInvalidConfiguration(f"{self.ref} shared requires sdl shared")
+        if Version(self.version).major != Version(self.dependencies["sdl"].ref.version).major:
+            raise ConanInvalidConfiguration(f"{self.ref} and sdl must have the same major version")
+
+    def build_requirements(self):
+        self.tool_requires("cmake/[>=3.16 <5]")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.cache_variables["SDLIMAGE_VENDORED"] = False
+        tc.cache_variables["SDLIMAGE_DEPS_SHARED"] = False
+        tc.cache_variables["SDLIMAGE_SAMPLES"] = False
+        tc.cache_variables["SDLIMAGE_STRICT"] = True
+        tc.cache_variables["SDLIMAGE_AVIF"] = self.options.with_avif
+        tc.cache_variables["SDLIMAGE_JPG"] = self.options.with_libjpeg
+        tc.cache_variables["SDLIMAGE_JXL"] = self.options.with_jxl
+        tc.cache_variables["SDLIMAGE_PNG"] = self.options.with_libpng
+        tc.cache_variables["SDLIMAGE_TIF"] = self.options.with_libtiff
+        tc.cache_variables["SDLIMAGE_WEBP"] = self.options.with_libwebp
+        tc.cache_variables["SDLIMAGE_BACKEND_WIC"] = self.options.get_safe("with_wic")
+        tc.cache_variables["SDLIMAGE_BACKEND_IMAGEIO"] = self.options.get_safe("with_imageio")
```

Comment:
```suggestion
```

I vote for removing these wic imageio as an option. Just checked other package managers, and they either don't expose or disable it. It's another option that we will be able to cover in the CI. Still, people can use it by Conan config without much effort.